.github/workflows/ci.yml

```
@@ -145,17 +149,36 @@ jobs:
         # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
         # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
         run: >
-            cmake -B build -S .
-#            -DCMAKE_CXX_COMPILER=${{ matrix.cpp_compiler }}
-#            -DCMAKE_C_COMPILER=${{ matrix.c_compiler }}
-#            -DCMAKE_BUILD_TYPE=${{ matrix.build_type }}
+          cmake -B build -S .
+          -D CMAKE_INSTALL_PREFIX=${{ steps.strings.outputs.build-output-dir }}/../install
 
```

It says you might need to add this to `aswf:` job
```suggestion
            - name: Set reusable strings
                    # Turn repeated input strings (such as the build output directory) into step outputs. These step outputs can be used throughout the workflow file.
                    id: strings
                    shell: bash
                    run: |
                      echo "build-output-dir=${{ github.workspace }}/build" >> "$GITHUB_OUTPUT"
```
on line 134